CONTRIBUTING.md

```diff
@@ -100,18 +101,26 @@ Test samples are kept in `/test/xxx/samples` folder.
 > PREREQUISITE: Install chromium via playwright by running `pnpm playwright install chromium`
 
 1. To run test, run `pnpm test`.
-1. To run test for a specific feature, you can use the `-g` (aka `--grep`) option. For example, to only run test involving transitions, run `pnpm test -- -g transition`.
+1. To run a particular test suite, use `pnpm test <suite-name>`, for example:
+
+   ```bash
+   pnpm test validator
+   ```
 
-##### Running solo test
+1. To filter tests _within_ a test suite, use `pnpm test <suite-name> -- -t <test-name>`, for example:
 
-1. To run only one test, rename the test sample folder to end with `.solo`. For example, to run the `test/js/samples/action` only, rename it to `test/js/samples/action.solo`.
-1. To run only one test suite, rename the test suite folder to end with `.solo`. For example, to run the `test/js` test suite only, rename it to `test/js.solo`.
-1. Remember to rename the test folder back. The CI will fail if there's a solo test.
+   ```bash
+   pnpm test validator -- -t a11y-alt-text
+   ```
 
 ##### Updating `.expected` files
 
-1. Tests suites like `css`, `js`, `server-side-rendering` asserts that the generated output has to match the content in the `.expected` file. For example, in the `js` test suites, the generated js code is compared against the content in `expected.js`.
-1. To update the content of the `.expected` file, run the test with `--update` flag. (`pnpm test --update`)
+1. Tests suites like `snapshot` and `parser` assert that the generated output matches the existing snapshot.
+1. To update these snapshots, run `UPDATE_SNAPSHOTS=true pnpm test`.
```

Comment:
Can we just do `pnpm run test -u`? I just delegate to `vitest` that way in `enhanced-img`, but not sure if there's a special reason why that won't work here

Reply:
No, this is using a completely different mechanism. I have no idea how Vitest snapshots work. If it makes sense for us to use them then we can, though it'd be a different PR

Reply:
ok i just read the vitest snapshot docs and yeah, i don't think it's appropriate for our `snapshot` tests. it would make life harder